pkg/gofr/handler_test.go

```diff
@@ -64,11 +64,12 @@ func TestHandler_ServeHTTP_Timeout(t *testing.T) {
 	w := httptest.NewRecorder()
 	r := httptest.NewRequest(http.MethodGet, "/", http.NoBody)
 
-	h := handler{requestTimeout: "1"}
+	// TODO: use a time.Duration for requestTimeout
```

Comment:
should we fix this TODO in this PR itself?

Reply:
done